mlbstatsapi/models/schedules/schedule.py

```diff
@@ -61,8 +74,8 @@ def get_games_with_status(self, abstractGameState = None,
                         gameIds.append(game.gamePk)
             return gameIds
 
-    def get_games_inProgress(self):
+    def get_games_inProgress(self) -> List[int]:
         return self.get_games_with_status(abstractGameState='Live')
 
-    def get_games_finnished(self):
+    def get_games_finnished(self) -> List[int]:
```

Comment:
finished* :)

Reply:
No, I want to specifically get games that are Finnish. Good catch again mate haha